doc/source/raysgd/raysgd_ptl.rst

```diff
@@ -0,0 +1,115 @@
+Pytorch Lightning with RaySGD
+==============================
+.. image:: /images/sgd_ptl.png
+  :align: center
+  :scale: 50 %
+
+
+RaySGD includes an integration with Pytorch Lightning's `LightningModule <https://pytorch-lightning.readthedocs.io/en/latest/lightning_module.html>`_.
+Easily take your existing ``LightningModule``, and use it with Ray SGD's ``TorchTrainer`` to take advantage of all of Ray SGD's distributed training features with minimal code changes.
+
+.. tip:: This LightningModule integration is currently under active development. If you encounter any bugs, please raise an issue on `Github <https://github.com/ray-project/ray/issues>`_!
+
+.. note:: Not all Pytorch Lightning features are supported. A full list of unsupported model hooks is listed down :ref:`below <_ptl-unsupported-features>`. Please post any feature requests on `Github <https://github.com/ray-project/ray/issues>`_ and we will get to it shortly!
+
+.. contents::
+    :local:
+
+Quick Start
+-----------
+Step 1: Define your ``LightningModule`` just like how you would with Pytorch Lightning.
+
+.. code-block:: python
+
+    from pytorch_lightning.core.lightning import LightningModule
+
+    class MyLightningModule(LightningModule):
+        ...
+
+Step 2: Use the ``TrainingOperator.from_ptl`` method to convert the ``LightningModule`` to a Ray SGD compatible ``LightningOperator``.
+
+.. code-block:: python
+
+    from ray.util.sgd.torch import TrainingOperator
+
+    MyLightningOperator = TrainingOperator.from_ptl(MyLightningModule)
+
+Step 3: Use the Operator with Ray SGD's ``TorchTrainer``, just like how you would normally. See :ref:`torch-guide` for a more full guide on ``TorchTrainer``.
+
+.. code-block:: python
+
+    import ray
+    from ray.util.sgd.torch import TorchTrainer
+
+    ray.init()
+    trainer = TorchTrainer(training_operator_cls=MyLightningOperator, num_workers=4, use_gpu=True)
+    train_stats = trainer.train()
+
+And that's it! For a more comprehensive guide, see the MNIST tutorial below.
```

Comment:
link ref to mnist tutorial

Reply:
I linked it. It's literally right below though.